### PR TITLE
guided setup

### PR DESCRIPTION
**Pull request by mboss37**
Extension of documentation for Guided Setup using Connected App approach.
See the following ticket that is being used to track the change request: https://www.mulesoft.org/jira/browse/EXCQ-4733#

### CODE REVIEW

**Review by wwarner-salesforce — DISMISSED** [review]

**Comment by wwarner-salesforce** [comment]
Thank you for this PR! Please do not merge yet. I'll get approval from Fer, then an editorial review, then merge.

File: modules/ROOT/pages/guided-setup.adoc
Comment (ferferrarazzo) [inline modules/ROOT/pages/guided-setup.adoc]: The Name must be [Anypoint]

File: modules/ROOT/pages/guided-setup.adoc
Comment (ferferrarazzo) [inline modules/ROOT/pages/guided-setup.adoc]: May be we can use this section: https://docs.mulesoft.com/api-community-manager/manual-setup#create-connected-app-in-anypoint

File: modules/ROOT/pages/guided-setup.adoc
Comment (ferferrarazzo) [inline modules/ROOT/pages/guided-setup.adoc]: May be we can use this section:
https://docs.mulesoft.com/api-community-manager/manual-setup#create-a-key-pair

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]
@wwarner-salesforce the content is perfect, may be we can reuse other pages, but we should definitely add this to the doc.

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: Fixed.

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: I checked and ensured that these instructions match the ones in that section.

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: I checked and ensured that these instructions match the ones in that section.

**Review by ferferrarazzo — DISMISSED** [review]

**Comment by wwarner-salesforce** [comment]
- [x] get confirmation from Fernando that the name should be `[Anypoint]` and not `Anypoint`
- [x] get confirmation from Fernando that the manual setup change is correct

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
Otherwise, ask an organization administrator to create the connected app for you, or perform these steps:
```

Comment (kbostrom) [reply modules/ROOT/pages/guided-setup.adoc]: Can you find a way to break up this task so that we don't have to go to three levels...especially since most of the user action is at the third level?

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: Consider combining so that we don't repeat "create in Salesforce."
Also, then we can end with a colon that points directly to the substeps (if you haven't found a way to promote them).

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
... Click *Save*.
```

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: Is this explanation necessary, considering that the following step...follows?

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
.. Create a connected app in Anypoint Platform:
```

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: Is the user typing in a name or selecting a name from a list?

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
... In the section *Grant types*, enable *JWT Bearer*.
```

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
... In the section *Redirect URIs*, add `+http://localhost+`.
```

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: nec?

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
... In the section *Who can use this application?*, select *Members of this organization only*.
```

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
... In the section *Scopes*, add *Background Access* and *Full Access*.
```

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]:
```suggestion
... Click *Save*.
```

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: Is all of this necessary if the audience is working in the UI? Won't they see the display? Does it matter to the user that the cert. is used instead of the secret? And why are we repeating "save" and mentioning "following steps" that come directly after?
Reevaluate this task presentation: the steps are nested too deeply and it seems like we are overexplaining the steps.

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc:56]: Is this text supposed to be followup to the step above it, or are the next two steps really...:

. If ...role, provide....
. If not, provide these app details:

File: modules/ROOT/pages/guided-setup.adoc
Comment (kbostrom) [inline modules/ROOT/pages/guided-setup.adoc]: These are really too long to be just one step apiece, further complicating the step nesting issue if we break them out. 
Consider reorganizing this information for the next cycle.

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: Done. The only third level steps are now the 6 brief steps for creating a key pair. Also updated line 28 per your comment.

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: Combined and added colon.

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: done

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: shortened

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: done

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: typing- rewrote to clarify

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: done

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: done

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: shortened

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: done

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: done

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: done

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: shortened and rewrote for clarity, with less nesting and less explanation

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc:56]: To "Authenticate and authorize the connected app to access your Anypoint organization", reader performs first option or second option. To prevent an additional level of nesting I've kept these as followup and not separate steps. This matches the structure of the step "Create a connected app in your Anypoint organization for Salesforce to use as a consumer system." and its two options.

File: modules/ROOT/pages/guided-setup.adoc
Comment (wwarner-salesforce) [reply modules/ROOT/pages/guided-setup.adoc]: Shortened and rewrote for clarity.

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**
